sdks/java/container/license_scripts/dep_urls_java.yaml

```diff
@@ -121,6 +121,9 @@ jsp-api:
 jsr166e:
   '1.1.0':
     license: "https://raw.githubusercontent.com/twitter-archive/jsr166e/master/LICENSE"
+junrar:
```

Comment:
Is there a reason for not pinning a newer version ? https://mvnrepository.com/artifact/com.github.junrar/junrar

Reply:
I pin the version that is actually used on the release branch. Note that we don't pin this on master (perhaps on master we already picked up a newer version).

Reply:
I didn't reproduce the error on master, so only pinning on the release branch.

Reply:
Sounds good.